build-deps.zsh

```diff
@@ -85,9 +85,9 @@ package() {
   if [[ ${PACKAGE_NAME} != 'qt'* ]] {
     log_status "Cleanup unnecessary files"
 
-    rm -rf lib/^(*.dylib|libajantv*|*.a|*.so*|*.lib|*.framework|*.dSYM|cmake)(N)
+    rm -rf lib/^(*.dylib|libajantv*|*.a|*.so*|*.lib|*.framework|*.dSYM|carla|cmake)(N)
```

Comment:
Unnecessary, as your libraries will either be available as as an archive, import library, or dynamic library (each of which is already excluded by the glob pattern).

Reply:
the libs are installed in $prefix/lib/carla, so this part is needed. the glob doesn't match against it otherwise.